syntax = "proto3";

message CreateShareRequest {
   string creator = 1;
   string name = 2;
   string id = 3; 
   int32 size = 4;  
}

message DeleteShareRequest {
   string id = 1;
   string creator = 2;
}

message Response {
   string msg = 1;
   int32 response_code = 2;
}

service EOS {
   rpc CreateShare(CreateShareRequest) returns (Response) {}
   rpc DeleteShare(DeleteShareRequest) returns (Response) {}
}

